### PR TITLE
[IOTDB-5704] Change default parameters for wal part

### PR DESCRIPTION
**Pull request by xingtanzjr**
## Description
Change default parameter values for wal part. See the changes below
```
sync_wal_delay_in_ms=1500
wal_buffer_size_in_byte=33554432
wal_file_size_threshold_in_byte=31457280
wal_buffer_queue_capacity=500
```

### CODE REVIEW

File: node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 1]
```diff
@@ -663,22 +663,22 @@ cluster_name=defaultCluster
 # A duration greater than 0 batches multiple wal fsync calls into one. This is useful when disks are slow or WAL write contention exists.
 # Notice: this value affects write performance significantly, values in the range of 0ms-10ms are recommended.
```
Comment (OneSizeFitsQuorum) [inline node-commons/src/assembly/resources/conf/iotdb-common.properties:664]: Change these comments?

File: node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 2]
```diff
@@ -663,22 +663,22 @@ cluster_name=defaultCluster
...
 # Datatype: long
-# fsync_wal_delay_in_ms=3
+# fsync_wal_delay_in_ms=1000
 
 # Buffer size of each wal node
 # If it's a value smaller than 0, use the default value 16 * 1024 * 1024 bytes (16MB).
```
Comment (OneSizeFitsQuorum) [inline node-commons/src/assembly/resources/conf/iotdb-common.properties:669]: Change these comments?

File: node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 3]
```diff
@@ -663,22 +663,22 @@ cluster_name=defaultCluster
...
+# wal_buffer_size_in_byte=33554432
 
 # Blocking queue capacity of each wal buffer, restricts maximum number of WALEdits cached in the blocking queue.
 # Datatype: int
-# wal_buffer_queue_capacity=50
+# wal_buffer_queue_capacity=500
 
 # Size threshold of each wal file
 # When a wal file's size exceeds this, the wal file will be closed and a new wal file will be created.
 # If it's a value smaller than 0, use the default value 10 * 1024 * 1024 (10MB).
```
Comment (OneSizeFitsQuorum) [inline node-commons/src/assembly/resources/conf/iotdb-common.properties:679]: Change these comments?

**Comment by OneSizeFitsQuorum** [comment]
It's seems that we should update the [docs](https://iotdb.apache.org/zh/UserGuide/Master/Reference/Common-Config-Manual.html#%E7%B3%BB%E7%BB%9F%E9%85%8D%E7%BD%AE%E9%A1%B9) too

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**